SPECIFICATION/README.md

```diff
@@ -0,0 +1,185 @@
+# DRAFTING PROCESS
+###### How to interact with THIS document
+1. Every Participant can `add`/`update`/`remove` his own ideas under **MICRO PROPOSALS**.
+* If you fully support a micro proposal of someone else, please add your name to support it. For example to copy (=support) proposal `foo bar baz` from user `xyz`, add your name behind it:
+  * foo bar baz [`@xyz`, `@yourname`]
+* **Please ask others questions about their micro proposals or propose to re-phrase/re-factor them into shorter proposals via pull request.**
+* Quick discussions can happen under github issues *(please quote __THIS__ document)*
+* Other changes should be made through **PULL REQUESTS** so that we can discuss
+
+
+
+# BASIC CONCEPT
+
+###### People
+1. `People` are unique real human beings
+
+###### Reputation
+1. Is given by `People` to `People`
+2. Every action that brings up/down `Reputation` of a person in the eyes of other persons should be reflected in the `People Network Reputation Journal`, thus `People` have the option to express it
+
+###### Network (=Engine)
+1. `Network` is how `people` can and want to interact
+* `Network` maintains a `Reputation Journal` (=**Reputation Points**)
+* `Network` maintains a `Rules Journal`
+* `Network` maintains an `Issue Journal`
```

Comment:
maybe people who back a task can put certain `task constraints` on it
* e.g. "i only back with 5 points if x other people do the same"